src/Cell.tsx

```diff
@@ -56,7 +55,6 @@ function Cell<R>({
   function handleCellDoubleClick(e: React.MouseEvent<HTMLDivElement>) {
     e.stopPropagation();
```

Comment:
Do we want to keep `e.stopPropagation()`?

Reply:
https://github.com/adazzle/react-data-grid/pull/1123
It originates from this PR, but I can't think of a reason to keep this, I'll remove it.